### DISCUSSION TITLE
Source AWS Cloud Watch

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#21421] [Q&A]
Does vector have a source that can use AWS Cloud watch as a source for metrics?

**Comment by jszwedko** [comment]
Hey! Not yet unfortunately, see https://github.com/vectordotdev/vector/issues/3383. I think you might be able to wire something together with AWS CloudWatch Metrics Streams -> AWS Firehose -> Vector, though.

**Reply by Manniac07** [reply]
Thank you we'll look into that
